pkg/capabilities/consensus/ocr3/capability.go

```diff
@@ -47,11 +47,16 @@ type capability struct {
 
 	transmitCh chan *response
 	newTimerCh chan *request
+
+	callbackChannelBufferSize int
 }
 
 var _ capabilityIface = (*capability)(nil)
 
-func newCapability(s *store, clock clockwork.Clock, requestTimeout time.Duration, encoderFactory EncoderFactory, lggr logger.Logger) *capability {
+var _ capabilities.ActionCapability = (*capability)(nil)
+
+func newCapability(s *store, clock clockwork.Clock, requestTimeout time.Duration, encoderFactory EncoderFactory, lggr logger.Logger,
+	callbackChannelBufferSize int) *capability {
```

Comment:
I wonder if we should create a dedicated OCR3CapabilityConfig struct at this point.